src/types/Todo.ts

```diff
@@ -1,6 +1,6 @@
 export interface Todo {
   id: number;
-  userId: number;
+  userId: number | undefined;
```

Comment:
you can replace userId to be optional param
```suggestion
  userId?: number;
```

Reply:
fine